src/traffic_layout/engine.cc

```diff
@@ -216,6 +217,17 @@ RunrootEngine::runroot_parse()
       fix_flag = true;
       continue;
     }
+    if (argument.substr(0, COPYSTYLE_WORD_LENGTH) == "--copy-style") {
+      std::string style = argument.substr(COPYSTYLE_WORD_LENGTH + 1);
+      if (style == "full") {
```

Comment:
Is this a case insensitive compare? Otherwise your usage message is steering people wrong.